packages/frontend-shared/src/warning/Warning.vue

```diff
@@ -64,6 +66,12 @@ let message = computed(() => {
     return [props.message, `        ${ props.details }`].join('\n\n')
   }
 
+  if (props.helpLinkHref) {
+    const learnMoreLabel = t('links.learnMoreButton')
+
+    return `${props.message} [${learnMoreLabel}](${props.helpLinkHref})`
+  }
+
```

Comment:
This is adding a very specific prop to this generic component.  Since this is  just formatting markdown in the message, you can just add the `Learn More` link to the message already formatted as Markdown. No need to modify this component.

Reply:
Yep, now that this is working using the markdown via @mike-plummer suggestion we can remove this. Fixed in e1f23d0e3c4a75c4e9ae4dff2b7247f43f7cfb66